spec/routing/hourly_rates_routing_spec.rb

```diff
@@ -19,30 +19,40 @@
 
 require 'spec_helper'
 
-describe HourlyRatesController, :type => :routing do
-  describe "routing" do
-    it { expect(get('/projects/blubs/hourly_rates/5')).to route_to(:controller => 'hourly_rates',
-                                                               :action => 'show',
-                                                               :project_id => 'blubs',
-                                                               :id => '5') }
+describe HourlyRatesController, type: :routing do
+  describe 'routing' do
+    it {
+      expect(get('/projects/blubs/hourly_rates/5')).to route_to(controller: 'hourly_rates',
+                                                                action: 'show',
+                                                                project_id: 'blubs',
+                                                                id: '5')
+    }
 
-    it { expect(get('/projects/blubs/hourly_rates/5/edit')).to route_to(:controller => 'hourly_rates',
-                                                                    :action => 'edit',
-                                                                    :project_id => 'blubs',
-                                                                    :id => '5') }
+    it {
+      expect(get('/projects/blubs/hourly_rates/5/edit')).to route_to(controller: 'hourly_rates',
+                                                                     action: 'edit',
+                                                                     project_id: 'blubs',
+                                                                     id: '5')
+    }
 
-    it { expect(get('/hourly_rates/5/edit')).to route_to(:controller => 'hourly_rates',
-                                                     :action => 'edit',
-                                                     :id => '5') }
+    it {
+      expect(get('/hourly_rates/5/edit')).to route_to(controller: 'hourly_rates',
+                                                      action: 'edit',
+                                                      id: '5')
+    }
 
-    it { expect(put('/projects/blubs/hourly_rates/5')).to route_to(:controller => 'hourly_rates',
-                                                               :action => 'update',
-                                                               :project_id => 'blubs',
-                                                               :id => '5') }
+    it {
+      expect(put('/projects/blubs/hourly_rates/5')).to route_to(controller: 'hourly_rates',
+                                                                action: 'update',
+                                                                project_id: 'blubs',
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.